eap74-openjdk17-image-stream.json

```diff
@@ -0,0 +1,135 @@
+            "openshift.io/provider-display-name": "Red Hat, Inc."
+        }
+    },
+    "items": [
+        {
+            "kind": "ImageStream",
+            "apiVersion": "image.openshift.io/v1",
+            "metadata": {
+                "name": "jboss-eap74-openjdk17-openshift",
+                "annotations": {
+                    "openshift.io/display-name": "JBoss EAP 7.4.0 with OpenJDK 17",
+                    "openshift.io/provider-display-name": "Red Hat, Inc.",
+                    "version": "7.4.0"
+                }
+            },
+            "labels": {
+                "xpaas": "7.4.0"
+            },
+            "spec": {
+                "tags": [
+                    {
+                        "name": "latest",
+                        "annotations": {
+                            "description": "The latest available build of JBoss EAP 7.4 with OpenJDK 17.",
+                            "iconClass": "icon-eap",
+                            "tags": "builder,eap,javaee,java,jboss,hidden",
+                            "supports": "eap:7.4,javaee:8,java:17",
+                            "sampleRepo": "https://github.com/jbossas/eap-quickstarts/",
+                            "sampleContextDir": "kitchensink",
+                            "sampleRef": "7.4.x",
+                            "version": "latest",
+                            "openshift.io/display-name": "JBoss EAP 7.4 with OpenJDK17"
+                        },
+                        "referencePolicy": {
+                            "type": "Local"
+                        },
+                        "from": {
+                            "kind": "DockerImage",
+                            "name": "registry.redhat.io/jboss-eap-7/eap74-openjdk17-openshift-rhel8:latest"
+                        }
+                    },
+                    {
+                        "name": "7.4.0",
+                        "annotations": {
+                            "description": "The latest available build of JBoss EAP 7.4.0 with OpenJDK 17.",
+                            "iconClass": "icon-eap",
+                            "tags": "builder,eap,javaee,java,jboss,hidden",
+                            "supports": "eap:7.4.0,javaee:8,java:17",
+                            "sampleRepo": "https://github.com/jbossas/eap-quickstarts/",
+                            "sampleContextDir": "kitchensink",
+                            "sampleRef": "7.4.x",
+                            "version": "latest",
+                            "openshift.io/display-name": "JBoss EAP 7.4.0 with OpenJDK17"
+                        },
+                        "referencePolicy": {
+                            "type": "Local"
+                        },
+                        "from": {
+                            "kind": "DockerImage",
+                            "name": "registry.redhat.io/jboss-eap-7/eap74-openjdk17-openshift-rhel8"
```

Comment:
do you have to use `registry.redhat.io/jboss-eap-7/eap74-openjdk17-openshift-rhel8:7.4.7` here?